packages/webvh/src/dids/didWebvhUtil.ts

```diff
@@ -0,0 +1,119 @@
+import { type AgentContext, Kms } from '@credo-ts/core'
+import { AbstractCrypto, type SigningOutput } from 'didwebvh-ts'
+
+export class DIDWebvhCrypto extends AbstractCrypto {
+  private agentContext: AgentContext
+
+  public constructor(agentContext: AgentContext) {
+    super({
+      verificationMethod: {
+        id: 'did:webvh:123',
+        controller: 'did:webvh:123',
+        type: 'Ed25519VerificationKey2020',
+        publicKeyMultibase: '123',
+        secretKeyMultibase: '123',
+      },
```

Comment:
Why do we set it to these sample values? Should this be optional in the `AbstractCrypto`?